src/components/Loader/PeopleList.tsx

```diff
@@ -0,0 +1,44 @@
+import classNames from 'classnames';
+import { Person } from '../../types';
+import { PersonLink } from './PersonLink';
+import { NO_PARENT } from '../../utils/constants';
+
+type Props = {
+  person: Person,
+  peopleSlug: string | undefined,
+};
+
+export const PeopleList: React.FC<Props> = ({ person, peopleSlug }) => {
```

Comment:
Looks like it's just for one person, you can name `PersonRow` or something similar